packages/components/psammead-play-button/src/index.stories.jsx

```diff
@@ -32,6 +32,21 @@ storiesOf('Components|PlayButton/Video', module)
       />
     ),
     { notes },
+  )
+  .add(
+    'video with guidance',
+    ({ service }) => (
+      <PlayButton
+        service={service}
+        title="Dog chases cat."
+        onClick={() => {}}
+        duration={text('duration', '2:30')}
+        durationSpoken={text('durationSpoken', '2 minutes 30 seconds')}
+        datetime={text('datetime', 'PT2M30S')}
+        guidanceMessage="Guidance: May contain strong language that may offend."
```

Comment:
```suggestion
        guidanceMessage={text(
          'guidanceMessage',
          'Guidance: May contain strong language that may offend.',
        )}
```

Reply:
For consistency, let's make this editable along with the other props in Storybook. 😄